### PR TITLE
Explain error C2373: '__pfnDliNotifyHook2': redefinition

### PR DESCRIPTION
**Pull request by jviotti**
Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

File: docs/README.hbs [hunk 1]
```diff
@@ -40,6 +40,20 @@ Run the test suite by doing:
 $ npm test
 ```
 
+Troubleshooting
+---------------
+
+### `error C2373: '__pfnDliNotifyHook2': redefinition`
+
+This error indicates that you're running a too old npm version. Upgrade by running:
```
Comment (lurch) [inline docs/README.hbs]: I think "...that the version of npm you're running is too old." might be more readable?

File: docs/README.hbs [hunk 2]
```diff
@@ -40,6 +40,20 @@ Run the test suite by doing:
...
+
+```sh
+npm install -g npm@latest
+```
+
+See the [following `node-gyp`
+issue](https://github.com/nodejs/node-gyp/issues/972) for more details.
```
Comment (lurch) [inline docs/README.hbs]: accidental line-break?

File: docs/README.hbs (hunk 1 above)
Comment (jviotti) [inline docs/README.hbs]: Fixed!

File: docs/README.hbs (hunk 2 above)
Comment (jviotti) [inline docs/README.hbs]: Fixed!

File: docs/README.hbs (hunk 1 above)
Comment (lurch) [inline docs/README.hbs]: You've now got "that that" ;-)

Comment (jviotti) [inline docs/README.hbs]: lol, fixed!

**Pull request merged**